gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/source/X509AuthSourceService.java

```diff
@@ -80,10 +97,22 @@ public boolean isValid(AuthSource authSource) {
      * @return true if client certificate is valid, false otherwise.
      */
     protected boolean isValid(X509Certificate clientCert) {
+        if (clientCert == null) {
+            return false;
+        }
+
+        final RequestContext context = RequestContext.getCurrentContext();
```

Comment:
seems like this method does not need to work with RequestContext, if that is true, could we move it to storeErrorHeader?

Reply:
fixed